### DISCUSSION TITLE
Problem deploying themes (Reclaim Hosting)

### DISCUSSION THREAD

**Discussion by mariape144** [CollectionBuilder/collectionbuilder.github.io#108] [Q&A]
Hi!
I have been workin with collection builder for a while using the csv version. When I run it in my environment (my pc) it goes great, I changed colors and banner configuration. When I bundle it shows me those changes but when I compress the _site file to upload to Reclaim Hosting (cpanel), the project works but doesn't show changes in colors, font, logo... Have I changed the colors in the wrong way (config-theme-colors.csv) or is there a problem with the bootstrap that generates the default configuration automatic?
Thank you!

**Comment by evanwill** [comment]
@mariape144 could you share a link to your project repository? 
It sounds like a CSS issue. One possibility is it might be that the links in the head aren't quite right, which would be if your url and baseurl in _config.yml don't match where you have it actually deployed?

**Reply by mariape144** [reply]
https://github.com/mariape144/Atlanticseascapes 
I'm not sure if it is a problem with Collection Builder or with cPanel, as my local hosting works perfectly. 
I appreciate your help!

**Reply by evanwill** [reply]
@mariape144 
First one detail, in your [_config.yml](https://github.com/mariape144/Atlanticseascapes/blob/main/_config.yml), I would add the full link to the `url` value, that is change `url: atlanticseascapesarchive.com` to `url: https://atlanticseascapesarchive.com`

As for the theme--is it working as expected now? I am seeing your customized colors and logo appear at <https://atlanticseascapesarchive.com/> (looks great!!)
If it is not appearing that way in your browser, you might need to do a hard refresh or private window to get the new CSS to load (the css file will stay cached in your browser, so it keeps using an old version). 

The thing that seems broken right now is the PDF files. It seems like your metadata is correct, however, the actual files are not in the "objects" folder on your server (or in the project on GitHub). You will want to upload the pdf files into the "objects" folder on your Reclaim server space with the rest of your built out website, see [object hosting notes in the docs](https://collectionbuilder.github.io/cb-docs/docs/objects/csv-objects/#object-hosting). You do not need to commit them into the GitHub repository, but if you would like to, check this [note about gitignore](https://collectionbuilder.github.io/cb-docs/docs/objects/csv-objects/#un-ignoring-the-objects-folder). 

Good luck!

**Comment by mariape144** [comment]
Thank you so much, Evan!
I followed your instructions and I was able to open the current version in a private window. It looks pretty nice, thank you!
I'm still uploading the PDFs to Reclaim, since the total size is 20 G, it should be over soon.

Thank you for the fast reply.
I'm presenting this at The Caribbean Digital Conference<http://Caribbean%20Digital%20Scholarship%20Collective>. I was afraid I was not going to be able to present the webpage but thanks to you I'm now able to do so!
Thank you also to Collection Builder Team for developing such a nice tool.

Maria Corredor
PhD Student History Department, Cornell University
________________________________
De: EvanWill ***@***.***>
Enviado: jueves, 7 de diciembre de 2023 3:28 p. m.
Para: CollectionBuilder/collectionbuilder.github.io ***@***.***>
Cc: mariape144 ***@***.***>; Mention ***@***.***>
Asunto: Re: [CollectionBuilder/collectionbuilder.github.io] Problem deploying themes (Reclaim Hosting) (Discussion #108)


@mariape144<https://github.com/mariape144>
First one detail, in your _config.yml<https://github.com/mariape144/Atlanticseascapes/blob/main/_config.yml>, I would add the full link to the url value, that is change url: atlanticseascapesarchive.com to url: https://atlanticseascapesarchive.com

As for the theme--is it working as expected now? I am seeing your customized colors and logo appear at https://atlanticseascapesarchive.com/ (looks great!!)
If it is not appearing that way in your browser, you might need to do a hard refresh or private window to get the new CSS to load (the css file will stay cached in your browser, so it keeps using an old version).

The thing that seems broken right now is the PDF files. It seems like your metadata is correct, however, the actual files are not in the "objects" folder on your server (or in the project on GitHub). You will want to upload the pdf files into the "objects" folder on your Reclaim server space with the rest of your built out website, see object hosting notes in the docs<https://collectionbuilder.github.io/cb-docs/docs/objects/csv-objects/#object-hosting>. You do not need to commit them into the GitHub repository, but if you would like to, check this note about gitignore<https://collectionbuilder.github.io/cb-docs/docs/objects/csv-objects/#un-ignoring-the-objects-folder>.

Good luck!
